spec/cypress/integration/schools/ViewPartnerships.feature

```diff
@@ -2,29 +2,23 @@ Feature: Induction tutors viewing partnerships
   Induction tutors should be able to view details of their chosen induction programme
 
   Background:
-   Given cohort was created with start_year "2021"
+    Given cohort was created with start_year "2021"
     And I am logged in as an "induction_coordinator"
     Then I should be on "choose programme advisory" page
-    And the page should be accessible
 
     When I click on "link" containing "Continue"
     Then I should be on "choose programme" page
-    And the page should be accessible
 
-  Scenario: View chosen programme
     When I click on "training provider" label
     And I click the submit button
     Then I should be on "schools" page
-    And the page should be accessible
 
     When I click on "link" containing "2021"
     Then I am on "2021 school cohorts" page
-    And "page body" should contain "Choose your induction programme"
-    And "page body" should contain "Confirm your training provider"
-    And "page body" should contain "Add early career teachers and mentors"
-    And the page should be accessible
```

Comment:
This was all duplicate from the Choose Programme spec. Ideally it would all be done using factory_bot in the the background like how ChooseMaterials.feature does it, but that felt outside the scope of this PR.